# Exporting `getToastQueue` to manage toasts
# Also, returning string from `addToast`
diff --git a/dist/chunk-LZV3PIXS.mjs b/dist/chunk-LZV3PIXS.mjs
index 511592d9ddfa61a4f3aada901b6a3e3ef6839527..8783445ff0d46f24a8db529280b414a2f2f96bcc 100644
--- a/dist/chunk-LZV3PIXS.mjs
+++ b/dist/chunk-LZV3PIXS.mjs
@@ -47,7 +47,7 @@ var addToast = ({ ...props }) => {
   if (!globalToastQueue) {
     return;
   }
-  globalToastQueue.add(props);
+  return globalToastQueue.add(props);
 };
 var closeAll = () => {
   if (!globalToastQueue) {
diff --git a/dist/index.d.mts b/dist/index.d.mts
index 76bfff69b963a6d765835945be3c3528a98a08f1..33eaad4d788da9677686c8e2995830c6fc1a8218 100644
--- a/dist/index.d.mts
+++ b/dist/index.d.mts
@@ -1,5 +1,5 @@
 export { default as Toast, ToastProps } from './toast.mjs';
-export { ToastProvider, addToast, closeAll } from './toast-provider.mjs';
+export { ToastProvider, addToast, closeAll, getToastQueue } from './toast-provider.mjs';
 export { useToast } from './use-toast.mjs';
 import '@heroui/system';
 import 'react/jsx-runtime';
diff --git a/dist/index.d.ts b/dist/index.d.ts
index 5eb5663f63b9466bd1dd0750599d855cccfcfc97..d16ed701f1389bde03c0f9d81f61ac6a03772906 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,5 +1,5 @@
 export { default as Toast, ToastProps } from './toast.js';
-export { ToastProvider, addToast, closeAll } from './toast-provider.js';
+export { ToastProvider, addToast, closeAll, getToastQueue } from './toast-provider.js';
 export { useToast } from './use-toast.js';
 import '@heroui/system';
 import 'react/jsx-runtime';
diff --git a/dist/index.js b/dist/index.js
index 6f7b044a26c4bff3b4edceb5eccdb59195e39451..b17d68ea3fc76b95c7e5d140c8e5936f84dec1b5 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -52,7 +52,8 @@ __export(index_exports, {
   ToastProvider: () => ToastProvider,
   addToast: () => addToast,
   closeAll: () => closeAll,
-  useToast: () => useToast
+  useToast: () => useToast,
+  getToastQueue: () => getToastQueue
 });
 module.exports = __toCommonJS(index_exports);
 
@@ -1158,5 +1159,6 @@ var closeAll = () => {
   ToastProvider,
   addToast,
   closeAll,
-  useToast
+  useToast,
+  getToastQueue
 });
diff --git a/dist/index.mjs b/dist/index.mjs
index f56408b6e3fd0291a70e36f30d1d6b922e0f901e..b47571ec070956943e10e509146eae95c6b9649a 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -2,7 +2,8 @@
 import {
   ToastProvider,
   addToast,
-  closeAll
+  closeAll,
+	getToastQueue
 } from "./chunk-LZV3PIXS.mjs";
 import "./chunk-FTGSR3HG.mjs";
 import {
@@ -16,5 +17,6 @@ export {
   ToastProvider,
   addToast,
   closeAll,
-  useToast
+  useToast,
+  getToastQueue
 };
diff --git a/dist/toast-provider.d.ts b/dist/toast-provider.d.ts
index 69d145f28742c3dc487f7ccdfc95b1e9299a1e5d..7c8a82dfdf1313bbc2eadb47f004b5485b2f942b 100644
--- a/dist/toast-provider.d.ts
+++ b/dist/toast-provider.d.ts
@@ -20,7 +20,7 @@ interface ToastProviderProps {
 }
 declare const getToastQueue: () => ToastQueue<ToastProps>;
 declare const ToastProvider: ({ placement, disableAnimation: disableAnimationProp, maxVisibleToasts, toastOffset, toastProps, regionProps, }: ToastProviderProps) => react_jsx_runtime.JSX.Element;
-declare const addToast: ({ ...props }: ToastProps & ToastOptions) => void;
+declare const addToast: ({ ...props }: ToastProps & ToastOptions) => string;
 declare const closeAll: () => void;
 
 export { ToastProvider, addToast, closeAll, getToastQueue };
